ReactiveCocoaTests/Swift/SignalSpec.swift

```diff
@@ -1484,5 +1484,263 @@ class SignalSpec: QuickSpec {
 				expect(error).to(equal(TestError.Default))
 			}
 		}
+		
+		describe("combinePrevious") {
+			
+			var sink: Signal<Int, NoError>.Observer!
+			var initialValue: Int!
+			var latestValues: (Int, Int)?
+			
+			beforeEach {
+				latestValues = nil
+				initialValue = 0
+				
+				let (signal, baseSink) = Signal<Int, NoError>.pipe()
+				sink = baseSink
+				signal |> combinePrevious(initialValue) |> observe(next: { latestValues = $0 })
+			}
+			
+			it("should forward the latest value with previous value") {
+				expect(latestValues).to(beNil())
+				
+				sendNext(sink, 1)
+				expect(latestValues?.0).to(equal(initialValue))
+				expect(latestValues?.1).to(equal(1))
+				
+				sendNext(sink, 2)
+				expect(latestValues?.0).to(equal(1))
+				expect(latestValues?.1).to(equal(2))
+			}
+			
+		}
+
+		describe("combineLatest") {
+			
+			var sinkA: Signal<Int, NoError>.Observer!
+			var sinkB: Signal<Int, NoError>.Observer!
+			var sinkC: Signal<Int, NoError>.Observer!
+			var sinkD: Signal<Int, NoError>.Observer!
+			var sinkE: Signal<Int, NoError>.Observer!
+			var sinkF: Signal<Int, NoError>.Observer!
+			var sinkG: Signal<Int, NoError>.Observer!
+			var sinkH: Signal<Int, NoError>.Observer!
+			var sinkI: Signal<Int, NoError>.Observer!
+			var sinkJ: Signal<Int, NoError>.Observer!
+			
+			var allSink: [Signal<Int, NoError>.Observer] = []
+			
+			var combinedValues: [Int]?
+			var completed: Bool!
+			var values: [Int]!
+			
+			beforeEach {
+				combinedValues = nil
+				completed = false
+				values = [0, 1, 2, 3, 4, 5, 6, 7, 8, 9]
+				
+				let (signalA, baseSinkA) = Signal<Int, NoError>.pipe()
+				let (signalB, baseSinkB) = Signal<Int, NoError>.pipe()
+				let (signalC, baseSinkC) = Signal<Int, NoError>.pipe()
+				let (signalD, baseSinkD) = Signal<Int, NoError>.pipe()
+				let (signalE, baseSinkE) = Signal<Int, NoError>.pipe()
+				let (signalF, baseSinkF) = Signal<Int, NoError>.pipe()
+				let (signalG, baseSinkG) = Signal<Int, NoError>.pipe()
+				let (signalH, baseSinkH) = Signal<Int, NoError>.pipe()
+				let (signalI, baseSinkI) = Signal<Int, NoError>.pipe()
+				let (signalJ, baseSinkJ) = Signal<Int, NoError>.pipe()
+				
+				sinkA = baseSinkA
+				sinkB = baseSinkB
+				sinkC = baseSinkC
+				sinkD = baseSinkD
+				sinkE = baseSinkE
+				sinkF = baseSinkF
+				sinkG = baseSinkG
+				sinkH = baseSinkH
+				sinkI = baseSinkI
+				sinkJ = baseSinkJ
+				
+				allSink = [sinkA, sinkB, sinkC, sinkD, sinkE, sinkF, sinkG, sinkH, sinkI, sinkJ]
+				
+				let combinedSignal = combineLatest(signalA, signalB, signalC, signalD, signalE, signalF, signalG, signalH, signalI, signalJ)
+				combinedSignal.observe(next: { combinedValues = [$0, $1, $2, $3, $4, $5, $6, $7, $8, $9] }, completed: { completed = true })
```

Comment:
Can you please expand the `next` and `completed` blocks over multiple lines, like the other tests in this file? (Same with `zip`.)